packages/core/src/utils/module.ts

```diff
@@ -1,9 +1,12 @@
+import { createRequire } from "node:module";
 import { normalizeImportPath } from "./path.js";
 
+const require = createRequire(import.meta.url);
+
 /**
  * Dead simple wrapper around import function to make it possible to mock it in the tests
  * @param path
  */
 export const importWrapper = async (path: string) => {
-  return import(normalizeImportPath(path));
+  return import(normalizeImportPath(require.resolve(path)));
```

Comment:
I would put `pathToFileURL` here. We need it not only on Windows but on Linux/Mac OS as well. Otherwise, it will throw `URI malformed` or `Cannot find module` on paths containing some special characters (e.g., `%`).

```suggestion
  return import(pathToFileURL(require.resolve(path)).href);
```